api/src/pcapi/alembic/versions/20240307T154918_3784f1bb8d9e_unique_program_for_an_educational_institution.py

```diff
@@ -0,0 +1,48 @@
+"""Unique program for an educational institution
+"""
+
+from alembic import op
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "3784f1bb8d9e"
+down_revision = "23fa7b1b692e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    with op.get_context().autocommit_block():
+        op.drop_index(
+            "ix_educational_institution_program_association_institutionId",
+            table_name="educational_institution_program_association",
+            if_exists=True,
+            postgresql_concurrently=True,
+        )
+        op.create_index(
+            op.f("ix_educational_institution_program_association_institutionId"),
+            "educational_institution_program_association",
+            ["institutionId"],
+            unique=True,
+            if_not_exists=True,
+            postgresql_concurrently=True,
+        )
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏si je comprends bien c'est une sorte de refresh de l'index ?

Reply:
C'est surtout pour le transformer un index `unique`, comme expliqué dans le commentaire dans [educational/models.py](https://github.com/pass-culture/pass-culture-main/pull/11142/files#diff-cf1132703a84e54e68a32af419be4d4c7b7e86f3408a3a4bf4fe2b4b52a88e00)